net/Sample/Views/Home/GroupedList.cshtml

```diff
@@ -10,14 +10,14 @@
                 store: DevExpress.data.AspNet.createStore({
                     loadUrl: "@Url.Action("Products", "Northwind")"
                 }),
-                group: "Category.CategoryName",
-                sort: "ProductName",
+                group: "category.categoryName",
```

Comment:
:cool: 
Did you have a chance to check insert/update operations. Do the work after this change?